packages/app/pages/api/models/[modelName]/validate/index.ts

```diff
@@ -0,0 +1,63 @@
+import { getLoggedInUser } from 'auth/user'
+import { strictValidateDocument } from 'documents/service'
+import { getModelWithWorkflow, userCanAccessModel } from 'models/service'
+import type { NextApiRequest, NextApiResponse } from 'next'
+import { respondAsJson } from 'utils/api'
+import { apiError, ErrorCode, HttpException } from 'utils/errors'
+import { safeParseJSON } from 'utils/json'
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
+    const modelName = decodeURIComponent(req.query.modelName.toString())
+    const user = await getLoggedInUser(req, res)
+
+    if (!userCanAccessModel(modelName, user)) {
+        return apiError(
+            new HttpException(
+                ErrorCode.ForbiddenError,
+                'User does not have access to the requested model'
+            ),
+            res
+        )
+    }
+
+    switch (req.method) {
+        //* GET returns the model's schema that will be used to valide the document.
+        case 'GET': {
```

Comment:
Can we use the existing schema endpoint for this? https://lb.gesdisc.eosdis.nasa.gov/meditor/api/models/Collection%20Metadata/schema

Would have to update the API docs as well

Reply:
Yep, I totally didn't see that this functionality had already been made in an existing endpoint. I'll remove the GET handler here and update the docs.